eng/pipelines/migration.yml

```diff
@@ -0,0 +1,118 @@
+      ExcludePaths: ''
+      IncludePaths: ''
+    - RepoOwner: Azure
+      RepoName: azure-sdk
+      ExcludePaths: "^./eng/common/*"
+      IncludePaths: ''
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-android
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/resources/session-records/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-c
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-cpp
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-go
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-ios
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-java
+      ExcludePaths: "^./sdk/|^./eng/common/|/resources/session-records/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-js
+      ExcludePaths: "^./sdk/|^./eng/common/|/recordings/node/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-net
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/SessionRecords/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-python
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/recordings/"
+      IncludePaths: "^./sdk/.*/platform-matrix.*.json$|^./sdk/.*/.*yml$"
+# The input parameter format is like:
+# - MigrateFrom: azsdk-pool-mms-win-2019-general
+#   MigrateTo: azsdk-pool-mms-win-2022-general
+# - MigrateFrom: windows-2019
+#   MigrateTo: windows-2022
+# - MigrateFrom: windows2019
+#   MigrateTo: windows2022
+# - MigrateFrom: MMS2019
+#   MigrateTo: MMS2022
+- name: MigrationMap
+  type: object
+  default: []
+- name: ShortForMigrateFrom
+  type: string
+  default: ''
+- name: ShortForMigrateTo
+  type: string
+  default: ''
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+jobs:
+  - job: Migration
+    pool:
+      name: azsdk-pool-mms-ubuntu-2004-general
+    variables:
+      - template: ./templates/variables/globals.yml
+    steps:
+      - ${{ each repo in parameters.Repos }}:
+        - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
+            SkipCheckoutNone: true
+            Repositories:
+              - Name: ${{ repo.RepoOwner }}/${{ repo.RepoName }}
+                WorkingDirectory: $(Build.SourcesDirectory)/docs/${{ repo.RepoName }}
```

Comment:
Why are we putting these under "docs" folder?

Reply:
I am trying to isolate tool script with the language SDK repo.
The structure is like:
- s
   - eng/
   - docs/
      - azure-sdk-for-java
      - azure-sdk-for-net
      ...

Reply:
Maybe it is better to call `langSDKs`. I worked on docs too much.

Reply:
Now the structure is like:
The structure is like:
- s
   - eng/
   - repos/
      - azure-sdk-for-java
      - azure-sdk-for-net
      ...